main.py

```diff
@@ -51,22 +71,26 @@ def main(cfg: DictConfig):
     logger.info(
         "Benchmarking countermeasured model for relative pre-training performance"
     )
-    pretrain_score.run_benchmark(
-        model_unadapted, wandb.config.pretrain, countermeasures=True, logger=logger
+    metrics.run_benchmark(
+        model_unadapted, countermeasures=True, logger=logger
     )
 
-    logger.info("Fine-tuning and recording results")
+    logger.info("Fine-tuning and recording results for unadapted model")
     ft_val_losses = fine_tuning.run_fine_tune(
         model_unadapted, wandb.config.finetune, logger
     )
 
-    logger.info("Calculating final unadaptability metrics")
-    fine_tuning.calculate_unadaptability_metrics(
-        ft_val_losses,
-        wandb.config.model.path,
-        logger,
+    # Just fine-tune the base model
+    logger.info("No unadaptability methods provided")
+    logger.info("Fine-tuning and recording results for base model")
+    ft_val_losses = fine_tuning.run_fine_tune(
+        model_base, wandb.config.finetune, logger
     )
 
+    # TODO -- Does this make sense for the fine-tune-only base model situation?
+    logger.info("Calculating final unadaptability metrics")
+    metrics.calculate_unadaptability_metrics(ft_val_losses, wandb.config.basename, logger)
```

Comment:
Yes, because it should be very fast to calculate the loss gap ratio, and it's a useful smoke test to make sure this is 0 for the base model.